File: JWP_API/JadeWesserPort/DTOs/StaffDTOs/StaffCreateDTO.cs
using JadeWesserPort.Domain;

namespace JadeWesserPort.DTOs.StaffDTOs;

public class StaffCreateDTO
{
    public int MecanographicNumber { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public int Phone { get; set; } 
    public StaffStatus Status { get; set; } = StaffStatus.Available;
    public List<string> QualificationCodes { get; set; } = [];
    public bool isActive = true;
}

File: Frontend/src/infrastructure/dtos/staff-member/staffCreateDto.ts
import type { StaffStatus } from "../../../domain/Enums/staffStatus";

export type staffCreateDto = {
  mecanographicNumber: number;
  name: string;
  email: string;
  phone: number;
  status: StaffStatus;
  qualificationCodes: string[];
  isActive: boolean;
};
